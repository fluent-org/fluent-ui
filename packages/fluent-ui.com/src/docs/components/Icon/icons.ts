export default [
  'Accident',
  'AccidentSolid',
  'ActionCenter',
  'ActionCenterAsterisk',
  'ActionCenterMirrored',
  'ActionCenterNotification',
  'ActionCenterNotificationMirrored',
  'ActionCenterQuiet',
  'ActionCenterQuietNotification',
  'AddRemoteDevice',
  'AddSurfaceHub',
  'AddTo',
  'AdjustHologram',
  'Airplane',
  'AirplaneSolid',
  'Annotation',
  'AppIconDefault',
  'Apps',
  'AreaChart',
  'ArrowDown8',
  'ArrowLeft8',
  'ArrowRight8',
  'ArrowUp8',
  'AspectRatio',
  'Asterisk',
  'AsteriskBadge12',
  'BackMirrored',
  'BackSpaceQWERTY',
  'BackSpaceQWERTYLg',
  'BackSpaceQWERTYMd',
  'BackSpaceQWERTYSm',
  'BackgroundToggle',
  'Badge',
  'Bank',
  'BarcodeScanner',
  'Battery0',
  'Battery1',
  'Battery10',
  'Battery2',
  'Battery3',
  'Battery4',
  'Battery5',
  'Battery6',
  'Battery7',
  'Battery8',
  'Battery9',
  'BatteryCharging0',
  'BatteryCharging1',
  'BatteryCharging10',
  'BatteryCharging2',
  'BatteryCharging3',
  'BatteryCharging4',
  'BatteryCharging5',
  'BatteryCharging6',
  'BatteryCharging7',
  'BatteryCharging8',
  'BatteryCharging9',
  'BatterySaver0',
  'BatterySaver1',
  'BatterySaver10',
  'BatterySaver2',
  'BatterySaver3',
  'BatterySaver4',
  'BatterySaver5',
  'BatterySaver6',
  'BatterySaver7',
  'BatterySaver8',
  'BatterySaver9',
  'BatteryUnknown',
  'Beta',
  'BidiLtr',
  'BidiRtl',
  'BlueLight',
  'Bluetooth',
  'BodyCam',
  'Brightness',
  'Broom',
  'BrushSize',
  'Bug',
  'BuildingEnergy',
  'BumperLeft',
  'BumperRight',
  'Bus',
  'BusSolid',
  'ButtonA',
  'ButtonB',
  'ButtonMenu',
  'ButtonView2',
  'ButtonX',
  'ButtonY',
  'Cafe',
  'CalculatorAddition',
  'CalculatorBackspace',
  'CalculatorDivide',
  'CalculatorEqualTo',
  'CalculatorMultiply',
  'CalculatorNegate',
  'CalculatorPercentage',
  'CalculatorSquareroot',
  'CalculatorSubtract',
  'CalendarMirrored',
  'CalendarSolid',
  'CallForwardInternational',
  'CallForwardInternationalMirrored',
  'CallForwardRoaming',
  'CallForwardRoamingMirrored',
  'CallForwarding',
  'CallForwardingMirrored',
  'CalligraphyFill',
  'CalligraphyPen',
  'CaretBottomRightSolidCenter8',
  'CaretDownSolid8',
  'CaretLeftSolid8',
  'CaretRight8',
  'CaretRightSolid8',
  'CaretUpSolid8',
  'CashDrawer',
  'Certificate',
  'CharacterAppearance',
  'ChatBubbles',
  'CheckList',
  'Checkbox14',
  'CheckboxComposite14',
  'CheckboxIndeterminateCombo',
  'CheckboxIndeterminateCombo14',
  'ChecklistMirrored',
  'ChevronDownMed',
  'ChevronLeftMed',
  'ChevronRightMed',
  'ChevronUpMed',
  'ChevronUpSmall',
  'ChineseBoPoMoFo',
  'ChineseChangjie',
  'ChinesePinyin',
  'ChinesePunctuation',
  'ChineseQuick',
  'ChipCardCreditCardReader',
  'ChromeAnnotate',
  'ChromeAnnotateContrast',
  'ChromeBack',
  'ChromeBackContrast',
  'ChromeBackContrastMirrored',
  'ChromeBackMirrored',
  'ChromeBackToWindow',
  'ChromeBackToWindowContrast',
  'ChromeClose',
  'ChromeCloseContrast',
  'ChromeFullScreen',
  'ChromeFullScreenContrast',
  'ChromeMaximize',
  'ChromeMaximizeContrast',
  'ChromeMinimize',
  'ChromeMinimizeContrast',
  'ChromeRestore',
  'ChromeRestoreContrast',
  'ChromeSwitch',
  'ChromeSwitchContast',
  'CircleFill',
  'CircleFillBadge12',
  'CircleRing',
  'CircleRingBadge12',
  'CityNext',
  'CityNext2',
  'ClearAllInk',
  'ClearAllInkMirrored',
  'Click',
  'ClipboardList',
  'ClipboardListMirrored',
  'Cloud',
  'CloudPrinter',
  'CloudSeach',
  'Code',
  'CollapseContent',
  'CollapseContentSingle',
  'CommaKey',
  'CommandPrompt',
  'Communications',
  'CompanionApp',
  'CompanionDeviceFramework',
  'Completed',
  'CompletedSolid',
  'Component',
  'Connect',
  'ConnectApp',
  'Connected',
  'Construction',
  'ConstructionCone',
  'ConstructionSolid',
  'ContactInfoMirrored',
  'ContactSolid',
  'Courthouse',
  'DMC',
  'DashKey',
  'DataSense',
  'DataSenseBar',
  'DateTime',
  'DateTimeMirrored',
  'DefaultAPN',
  'DefenderApp',
  'DefenderBadge12',
  'Design',
  'DetachablePC',
  'DevUpdate',
  'DeveloperTools',
  'DeviceDiscovery',
  'DeviceLaptopNoPic',
  'DeviceLaptopPic',
  'DeviceMonitorLeftPic',
  'DeviceMonitorNoPic',
  'DeviceMonitorRightPic',
  'Devices',
  'Devices3',
  'Devices4',
  'Diagnostic',
  'Dial1',
  'Dial10',
  'Dial11',
  'Dial12',
  'Dial13',
  'Dial14',
  'Dial15',
  'Dial16',
  'Dial2',
  'Dial3',
  'Dial4',
  'Dial5',
  'Dial6',
  'Dial7',
  'Dial8',
  'Dial9',
  'DialShape1',
  'DialShape2',
  'DialShape3',
  'DialShape4',
  'DialUp',
  'Dialpad',
  'Dictionary',
  'DictionaryAdd',
  'DictionaryCloud',
  'DirectAccess',
  'DisconnectDisplay',
  'Dock',
  'DoublePinyin',
  'DownShiftKey',
  'DownloadMap',
  'Dpad',
  'Draw',
  'DrawSolid',
  'DrivingMode',
  'Drop',
  'DullSound',
  'DullSoundKey',
  'EMI',
  'Education',
  'EmojiSwatch',
  'EmojiTabCelebrationObjects',
  'EmojiTabFavorites',
  'EmojiTabFoodPlants',
  'EmojiTabPeople',
  'EmojiTabSmilesAnimals',
  'EmojiTabSymbols',
  'EmojiTabTextSmiles',
  'EmojiTabTransitPlaces',
  'EndPoint',
  'EndPointSolid',
  'EnglishPunctuation',
  'EraseTool',
  'EraseToolFill',
  'EraseToolFill2',
  'Error',
  'ErrorBadge',
  'ErrorBadge12',
  'Ethernet',
  'EthernetError',
  'EthernetWarning',
  'ExploreContent',
  'ExploreContentSingle',
  'Export',
  'ExportMirrored',
  'Eyedropper',
  'FastForward',
  'Favicon',
  'FavoriteList',
  'Feedback',
  'FeedbackApp',
  'Ferry',
  'FerrySolid',
  'FileExplorer',
  'FileExplorerApp',
  'FingerInking',
  'Fingerprint',
  'Flashlight',
  'FlickDown',
  'FlickLeft',
  'FlickRight',
  'FlickUp',
  'Folder',
  'FolderHorizontal',
  'FolderOpen',
  'ForwardMirrored',
  'ForwardSm',
  'Frigid',
  'FullAlpha',
  'FullCircleMask',
  'FullHiragana',
  'FullKatakana',
  'Game',
  'GameConsole',
  'GlobalNavigationButton',
  'GridView',
  'GripperBarHorizontal',
  'GripperBarVertical',
  'GripperResize',
  'GripperResizeMirrored',
  'GripperTool',
  'Groceries',
  'Group',
  'GroupList',
  'GuestUser',
  'HalfAlpha',
  'HalfDullSound',
  'HalfKatakana',
  'HalfStarLeft',
  'HalfStarRight',
  'Handwriting',
  'HardDrive',
  'HeadlessDevice',
  'Headphone',
  'Headphone0',
  'Headphone1',
  'Headphone2',
  'Headphone3',
  'Headset',
  'Health',
  'HeartBroken',
  'HighlightFill',
  'HighlightFill2',
  'History',
  'HomeGroup',
  'HomeSolid',
  'HorizontalTabKey',
  'IBeam',
  'IBeamOutline',
  'ImageExport',
  'ImportantBadge12',
  'InPrivate',
  'IncidentTriangle',
  'Info',
  'Info2',
  'InfoSolid',
  'InkingCaret',
  'InkingColorFill',
  'InkingColorOutline',
  'InkingTool',
  'InkingToolFill',
  'InkingToolFill2',
  'Input',
  'InsiderHubApp',
  'InternetSharing',
  'Japanese',
  'JpnRomanji',
  'JpnRomanjiLock',
  'JpnRomanjiShift',
  'JpnRomanjiShiftLock',
  'Key12On',
  'KeyboardBrightness',
  'KeyboardDismiss',
  'KeyboardFull',
  'KeyboardLeftHanded',
  'KeyboardLowerBrightness',
  'KeyboardOneHanded',
  'KeyboardRightHanded',
  'KeyboardShortcut',
  'Korean',
  'LEDLight',
  'Label',
  'LangJPN',
  'LanguageChs',
  'LanguageCht',
  'LanguageJpn',
  'LanguageKor',
  'LaptopSelected',
  'LargeErase',
  'Leaf',
  'LeaveChatMirrored',
  'LeftArrowKeyTime0',
  'LeftDoubleQuote',
  'LeftQuote',
  'LeftStick',
  'Lexicon',
  'Light',
  'Lightbulb',
  'LightningBolt',
  'LineDisplay',
  'List',
  'ListMirrored',
  'LockFeedback',
  'LockScreenGlance',
  'LockscreenDesktop',
  'LowerBrightness',
  'MagStripeReader',
  'MailBadge12',
  'MapCompassBottom',
  'MapCompassTop',
  'MapDirections',
  'MapLayers',
  'Marker',
  'Marquee',
  'Media',
  'MediaStorageTower',
  'Megaphone',
  'MicClipping',
  'MicError',
  'MicOff',
  'MicOn',
  'MicSleep',
  'MicrophoneListening',
  'MiracastLogoLarge',
  'MiracastLogoSmall',
  'MobActionCenter',
  'MobAirplane',
  'MobBattery0',
  'MobBattery1',
  'MobBattery10',
  'MobBattery2',
  'MobBattery3',
  'MobBattery4',
  'MobBattery5',
  'MobBattery6',
  'MobBattery7',
  'MobBattery8',
  'MobBattery9',
  'MobBatteryCharging0',
  'MobBatteryCharging1',
  'MobBatteryCharging10',
  'MobBatteryCharging2',
  'MobBatteryCharging3',
  'MobBatteryCharging4',
  'MobBatteryCharging5',
  'MobBatteryCharging6',
  'MobBatteryCharging7',
  'MobBatteryCharging8',
  'MobBatteryCharging9',
  'MobBatterySaver0',
  'MobBatterySaver1',
  'MobBatterySaver10',
  'MobBatterySaver2',
  'MobBatterySaver3',
  'MobBatterySaver4',
  'MobBatterySaver5',
  'MobBatterySaver6',
  'MobBatterySaver7',
  'MobBatterySaver8',
  'MobBatterySaver9',
  'MobBatteryUnknown',
  'MobBluetooth',
  'MobCallForwarding',
  'MobCallForwardingMirrored',
  'MobDrivingMode',
  'MobLocation',
  'MobQuietHours',
  'MobSIMLock',
  'MobSIMMissing',
  'MobSignal1',
  'MobSignal2',
  'MobSignal3',
  'MobSignal4',
  'MobSignal5',
  'MobWifi1',
  'MobWifi2',
  'MobWifi3',
  'MobWifi4',
  'MobWifiHotspot',
  'MobeSIM',
  'MobeSIMBusy',
  'MobeSIMLocked',
  'MobeSIMNoProfile',
  'MobileLocked',
  'MobileSelected',
  'MobileTablet',
  'Mouse',
  'Movies',
  'MultimediaDMP',
  'MultimediaDMS',
  'MultimediaDVR',
  'MultimediaPMP',
  'MusicAlbum',
  'MusicNote',
  'MyNetwork',
  'NUIFPContinueSlideAction',
  'NUIFPContinueSlideHand',
  'NUIFPPressAction',
  'NUIFPPressHand',
  'NUIFPPressRepeatAction',
  'NUIFPPressRepeatHand',
  'NUIFPRollLeftAction',
  'NUIFPRollLeftHand',
  'NUIFPRollRightHand',
  'NUIFPRollRightHandAction',
  'NUIFPStartSlideAction',
  'NUIFPStartSlideHand',
  'NUIFace',
  'NUIIris',
  'Narrator',
  'NarratorForward',
  'NarratorForwardMirrored',
  'Network',
  'NetworkAdapter',
  'NetworkPrinter',
  'NetworkSharing',
  'NetworkTower',
  'OEM',
  'OpenFolderHorizontal',
  'OutlineHalfStarLeft',
  'OutlineHalfStarRight',
  'OutlineQuarterStarLeft',
  'OutlineQuarterStarRight',
  'OutlineStarLeftHalf',
  'OutlineStarRightHalf',
  'OutlineThreeQuarterStarLeft',
  'OutlineThreeQuarterStarRight',
  'PINPad',
  'PLAP',
  'Package',
  'PageLeft',
  'PageRight',
  'PaginationDotOutline10',
  'PaginationDotSolid10',
  'PanMode',
  'ParkingLocation',
  'ParkingLocationMirrored',
  'ParkingLocationSolid',
  'PartyLeader',
  'PasswordKeyHide',
  'PasswordKeyShow',
  'PauseBadge12',
  'PaymentCard',
  'PenPalette',
  'PenPaletteMirrored',
  'PenWorkspace',
  'PenWorkspaceMirrored',
  'Pencil',
  'PencilFill',
  'PeriodKey',
  'PersonalFolder',
  'Personalize',
  'Photo',
  'Photo2',
  'PieSingle',
  'PinFill',
  'Pinned',
  'PinnedFill',
  'PinyinIMELogo',
  'Play36',
  'PlayBadge12',
  'PlaybackRate1x',
  'PlaybackRateOther',
  'PlayerSettings',
  'PointErase',
  'PointEraseMirrored',
  'PoliceCar',
  'PowerButton',
  'Printer3D',
  'PrintfaxPrinterFile',
  'Process',
  'Processing',
  'ProgressRingDots',
  'Project',
  'Projector',
  'Protractor',
  'ProvisioningPackage',
  'PuncKey',
  'PuncKey0',
  'PuncKey1',
  'PuncKey2',
  'PuncKey3',
  'PuncKey4',
  'PuncKey5',
  'PuncKey6',
  'PuncKey7',
  'PuncKey8',
  'PuncKey9',
  'PuncKeyLeftBottom',
  'PuncKeyRightBottom',
  'Puzzle',
  'QWERTYOff',
  'QWERTYOn',
  'QuarentinedItems',
  'QuarentinedItemsMirrored',
  'QuarterStarLeft',
  'QuarterStarRight',
  'QuickNote',
  'QuietHours',
  'QuietHoursBadge12',
  'RadioBtnOff',
  'RadioBtnOn',
  'RadioBullet',
  'RadioBullet2',
  'ReadingList',
  'ReceiptPrinter',
  'Relationship',
  'RememberedDevice',
  'Reminder',
  'ReminderFill',
  'RemoveFrom',
  'Replay',
  'ReplyMirrored',
  'ReportDocument',
  'ResetDevice',
  'ResetDrive',
  'ResizeMouseLarge',
  'ResizeMouseMedium',
  'ResizeMouseMediumMirrored',
  'ResizeMouseSmall',
  'ResizeMouseSmallMirrored',
  'ResizeMouseTall',
  'ResizeMouseTallMirrored',
  'ResizeMouseWide',
  'ResizeTouchLarger',
  'ResizeTouchNarrower',
  'ResizeTouchNarrowerMirrored',
  'ResizeTouchShorter',
  'ResizeTouchSmaller',
  'ReturnKey',
  'ReturnKeyLg',
  'ReturnKeySm',
  'RevToggleKey',
  'Rewind',
  'RightArrowKeyTime0',
  'RightArrowKeyTime1',
  'RightArrowKeyTime2',
  'RightArrowKeyTime3',
  'RightArrowKeyTime4',
  'RightDoubleQuote',
  'RightQuote',
  'RightStick',
  'Ringer',
  'RingerBadge12',
  'RingerSilent',
  'RoamingDomestic',
  'RoamingInternational',
  'Robot',
  'Rotate',
  'RotateMapLeft',
  'RotateMapRight',
  'RotationLock',
  'Ruler',
  'SDCard',
  'SIPMove',
  'SIPRedock',
  'SIPUndock',
  'SaveCopy',
  'ScreenTime',
  'ScrollMode',
  'ScrollUpDown',
  'SearchAndApps',
  'SendFill',
  'SendFillMirrored',
  'SendMirrored',
  'Sensor',
  'SettingsBattery',
  'SettingsDisplaySound',
  'Share',
  'ShareBroadband',
  'Shield',
  'ShoppingCart',
  'SignalBars1',
  'SignalBars2',
  'SignalBars3',
  'SignalBars4',
  'SignalBars5',
  'SignalError',
  'SignalNotConnected',
  'SignalRoaming',
  'SignatureCapture',
  'SkipBack10',
  'SkipForward30',
  'SliderThumb',
  'SlowMotionOn',
  'SmallErase',
  'Smartcard',
  'SmartcardVirtual',
  'SpatialVolume0',
  'SpatialVolume1',
  'SpatialVolume2',
  'SpatialVolume3',
  'Speakers',
  'Speech',
  'SpeedHigh',
  'SpeedMedium',
  'SpeedOff',
  'StartPoint',
  'StartPointSolid',
  'StatusCheckmark7',
  'StatusCircle',
  'StatusCircle7',
  'StatusCircleBlock',
  'StatusCircleBlock2',
  'StatusCircleCheckmark',
  'StatusCircleErrorX',
  'StatusCircleExclamation',
  'StatusCircleInfo',
  'StatusCircleInner',
  'StatusCircleLeft',
  'StatusCircleOuter',
  'StatusCircleQuestionMark',
  'StatusCircleRing',
  'StatusCircleSync',
  'StatusConnecting1',
  'StatusConnecting2',
  'StatusDataTransfer',
  'StatusDataTransferVPN',
  'StatusDualSIM1',
  'StatusDualSIM1VPN',
  'StatusDualSIM2',
  'StatusDualSIM2VPN',
  'StatusError',
  'StatusErrorCircle7',
  'StatusErrorFull',
  'StatusErrorLeft',
  'StatusExclamationCircle7',
  'StatusPause7',
  'StatusSGLTE',
  'StatusSGLTECell',
  'StatusSGLTEDataVPN',
  'StatusTriangle',
  'StatusTriangleExclamation',
  'StatusTriangleInner',
  'StatusTriangleLeft',
  'StatusTriangleOuter',
  'StatusUnsecure',
  'StatusVPN',
  'StatusWarning',
  'StatusWarningLeft',
  'StockDown',
  'StockUp',
  'StopPoint',
  'StopPointSolid',
  'Stopwatch',
  'StorageNetworkWireless',
  'StorageOptical',
  'StorageTape',
  'Streaming',
  'StreamingEnterprise',
  'StreetsideSplitExpand',
  'StreetsideSplitMinimize',
  'StrokeErase',
  'StrokeErase2',
  'StrokeEraseMirrored',
  'Subtitles',
  'SubtitlesAudio',
  'SurfaceHub',
  'Sustainable',
  'Swipe',
  'SwipeRevealArt',
  'SwitchUser',
  'SyncBadge12',
  'SyncError',
  'System',
  'TVMonitor',
  'TVMonitorSelected',
  'Tablet',
  'TabletMode',
  'TabletSelected',
  'TapAndSend',
  'TaskView',
  'TaskViewExpanded',
  'TaskViewSettings',
  'TaskbarPhone',
  'ThisPC',
  'ThoughtBubble',
  'ThreeQuarterStarLeft',
  'ThreeQuarterStarRight',
  'Tiles',
  'TiltDown',
  'TiltUp',
  'TimeLanguage',
  'ToggleBorder',
  'ToggleFilled',
  'ToggleThumb',
  'TollSolid',
  'ToolTip',
  'Touch',
  'Touchpad',
  'Touchscreen',
  'Trackers',
  'TrackersMirrored',
  'TrafficCongestionSolid',
  'TrafficLight',
  'Train',
  'TrainSolid',
  'TreeFolderFolder',
  'TreeFolderFolderFill',
  'TreeFolderFolderOpen',
  'TreeFolderFolderOpenFill',
  'TriggerLeft',
  'TriggerRight',
  'USB',
  'USBSafeConnect',
  'UnderscoreSpace',
  'Unit',
  'UpArrowShiftKey',
  'UpShiftKey',
  'UpdateRestore',
  'UserAPN',
  'VPN',
  'Vibrate',
  'VideoSolid',
  'Volume0',
  'Volume1',
  'Volume2',
  'Volume3',
  'VolumeBars',
  'Walk',
  'WalkSolid',
  'Warning',
  'Webcam2',
  'Website',
  'Wheel',
  'Wifi',
  'Wifi1',
  'Wifi2',
  'Wifi3',
  'WifiAttentionOverlay',
  'WifiCall0',
  'WifiCall1',
  'WifiCall2',
  'WifiCall3',
  'WifiCall4',
  'WifiCallBars',
  'WifiError0',
  'WifiError1',
  'WifiError2',
  'WifiError3',
  'WifiError4',
  'WifiEthernet',
  'WifiHotspot',
  'WifiWarning0',
  'WifiWarning1',
  'WifiWarning2',
  'WifiWarning3',
  'WifiWarning4',
  'WindDirection',
  'WiredUSB',
  'WirelessUSB',
  'Work',
  'WorkSolid',
  'XboxOneConsole',
  'ZoomMode'
]
