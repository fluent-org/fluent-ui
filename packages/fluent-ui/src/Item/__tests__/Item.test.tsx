import * as React from 'react'
import * as renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Connected } from '@fluent-ui/icons'

import { ThemeProvider, Item } from '../..'

describe('Item', (): void => {
  const theme = {}
  test('basic', (): void => {
    const component = renderer.create(
      <ThemeProvider theme={theme}>
        <Item prefix={<Connected />}>Option</Item>
      </ThemeProvider>
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('without children', (): void => {
    const component = renderer.create(
      <ThemeProvider theme={theme}>
        <Item prefix={<Connected />} />
      </ThemeProvider>
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
