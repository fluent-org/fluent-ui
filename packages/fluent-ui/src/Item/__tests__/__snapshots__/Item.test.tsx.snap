// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item should be support prefix 1`] = `
<div
  class="FBox-root FBox-system FBox-system-0 FItem-root"
>
  <div
    class="FItem-prefixRoot"
  >
    <svg
      data-testid="icon"
      style="width: 1.3em; height: 1.3em; display: inline-block; font-size: inherit; fill: currentColor;"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M3 4h18v2H3V4zm0 7h18v2H3v-2zm0 7h18v2H3v-2z"
      />
    </svg>
  </div>
  <div
    class="FItem-text FItem-textExpanded FItem-textHasPrefix"
  >
    Hello World
  </div>
</div>
`;

exports[`Item should be support prefix without children 1`] = `
<div>
  <div
    class="FBox-root FBox-system FBox-system-0 FItem-root"
  >
    <div
      class="FItem-prefixRoot"
    >
      <svg
        data-testid="icon"
        style="width: 1.3em; height: 1.3em; display: inline-block; font-size: inherit; fill: currentColor;"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M3 4h18v2H3V4zm0 7h18v2H3v-2zm0 7h18v2H3v-2z"
        />
      </svg>
    </div>
    <div
      class="FItem-text FItem-textExpanded FItem-textNoChildren FItem-textHasPrefix"
    />
  </div>
</div>
`;
